### ISSUE TITLE
auto-calorie calculation

### ISSUE THREAD

**Issue by d4rkd0s** [grepsedawk/planmy.hike#3]
![image](https://github.com/user-attachments/assets/366c89ea-8c77-4470-92a8-3ee24b711a3f)

After start miles are entered, 
and after end miles are entered,
days should be entered next,
then an auto-calculation can be performed for cals/day based on selecting predefined settings for,
(with Body weight and RMR) you can perform this calc.

Just a start of an idea 🤷

**Comment by grepsedawk** [comment]
Yes! Definitely within the plan. I also, additionally, want to allow for advanced features that can help adjust to personal needs vs benchmark/purely calculated. I have a lot of undocumented future features, and this is definitely on my mind for a "post-MVP" add. Thanks for making the issue/start of documentation for me!!
